frontend/src/components/Board/Settings/index.tsx

```diff
@@ -149,6 +159,40 @@ const BoardSettings = ({ isOpen, setIsOpen }: Props) => {
 		);
 	};
 
+	const configurationSettings = (
+		title: string,
+		text: string,
+		isChecked: boolean,
+		handleCheckedChanage: any,
+		child?: any
```

Comment:
The `children`  in React can be anything, so the best way to type it is using `ReactNode`:
```suggestion
		child?: ReactNode
```